src/test.ts

```diff
@@ -10,7 +10,7 @@ import * as Retraced from "./";
       projectId: "aaaahahahahahahaha",
     });
 
-    expect(garbo).to.exist;
+    console.log(expect(garbo).to.exist);
```

Comment:
remove console.log

Reply:
There were linking errors so I added logs at the end of tests

Reply:
linking errors as in?

Reply:
Lint*

Reply:
aah, let me check.